### PR TITLE
Refresh how-to-write-text-to-a-file.md

### PR DESCRIPTION
**Pull request by v-thepet**
## Summary

Describe your changes here.
Refresh file, validate C# code, consolidate code snippets, update terminology.

Fixes #Issue_Number (if available)
See USER STORY 1374049 and Task https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1385012

Associated samples PRs:
https://github.com/dotnet/samples/pull/543
https://github.com/dotnet/samples/pull/579

### CODE REVIEW

File: includes/win8-appname-long-md.md
Comment (mairaw) [inline includes/win8-appname-long-md.md]: I wouldn't change the content of this include file since it's probably used in other places to refer to Windows 8.x store. Instead, just change right on the text where the include file is used to say UWP.

**Review by mairaw — COMMENTED** [review]
A quick comment for now... I'll review this more thoroughly once I'm back to the office next week.

File: docs/standard/io/how-to-write-text-to-a-file.md
Comment (rpetrusha) [inline docs/standard/io/how-to-write-text-to-a-file.md]: I'd delete the sentence about the *My Documents* folder. Though true on Windows, it's not true on other platforms or in the C# REPL.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks, @v-thepet. Your changes look good. I've included a number of additional suggestions, though.

File: includes/win8-appname-long-md.md
Comment (v-thepet) [reply includes/win8-appname-long-md.md]: @mairaw I've removed the include file from the PR. We're actually now removing all mention of UWP/WIndows Store apps from this set of docs. (I don't understand why this comment that was made 7 days ago only yesterday appeared for me...!)

**Comment by rpetrusha** [comment]
Closing and reopening to begin new build after dotnet/samples#543 was merged.

**Comment by v-thepet** [comment]
@rpetrusha - The new code sample links are in the topic now - please check. Thanks.

File: docs/standard/io/how-to-write-text-to-a-file.md
Comment (rpetrusha) [inline docs/standard/io/how-to-write-text-to-a-file.md]: This example requires C# 7.1... --> The C# example requires C# 7.1...

File: docs/standard/io/how-to-write-text-to-a-file.md
Comment (rpetrusha) [inline docs/standard/io/how-to-write-text-to-a-file.md]: The VB code doesn't compile, since Async Main isn't supported. The code has to be:
```
Imports System.IO

Public Module Example
   Public Sub Main()
      WriteTextAsync()
   End Sub

   Async Sub WriteTextAsync() 
       ' Set a variable to the Documents path.
       Dim docPath As String = Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments)

       ' Write the text asynchronously to a new file named "WriteTextAsync.txt".
        Using outputFile As New StreamWriter(Path.Combine(docPath, Convert.ToString("WriteTextAsync.txt")))
           Await outputFile.WriteAsync("This is a sentence.")
       End Using
   End Sub
End Module
```

**Review by rpetrusha — COMMENTED** [review]
This topic is vastly improved, @v-thepet.
 
I've left one comment about this topic. Also, in addition to one VB example not compiling, I have a couple of comments about the code:
- You can remove `Imports System` from the VB examples; System is imported by default.
- It would be useful to include the output to the file as a comment in each example. (For example: `// The file created by this example has the following contents:
//    This is a sentence.

**Comment by v-thepet** [comment]
@rpetrusha - Content change is made and the code changes are in https://github.com/dotnet/samples/pull/579 - please review. Thanks.

**Comment by rpetrusha** [comment]
Closing and reopening to begin new build.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks, @v-thepet. I've made some minor changes and approved the PR, so I'll merge this after the build completes.

**Comment by v-thepet** [comment]
@rpetrusha - Looks great to me, thanks!

**Pull request merged**
